koku/api/provider/models.py

```diff
@@ -109,8 +109,8 @@ class Meta:
     CLOUD_PROVIDER_LIST = [choice[0] for choice in CLOUD_PROVIDER_CHOICES]
 
     uuid = models.UUIDField(default=uuid4, primary_key=True)
-    name = models.CharField(max_length=256, null=False)
-    type = models.CharField(max_length=50, null=False, choices=PROVIDER_CHOICES, default=PROVIDER_AWS)
+    name = models.TextField(null=False)
+    type = models.TextField(null=False, choices=PROVIDER_CHOICES, default=PROVIDER_AWS)
```

Comment:
`max_length` is required to avoid DDoS attack (according to a report we got from the security team in Red Hat).